xml/System.Net.WebSockets/WebSocket.xml

```diff
@@ -810,12 +810,12 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="messageType">To be added.</param>
-        <param name="endOfMessage">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">The buffer to be sent over the connection.</param>
+        <param name="messageType">Indicates whether the application is sending a binary or text message.</param>
+        <param name="endOfMessage">Indicates whether the data in "buffer" is the last part of a message.</param>
```

Comment:
```suggestion
        <param name="endOfMessage">
          <see langword="true" /> to indicates the data in "buffer" is the last part of a message; otherwise, <see langword="false" />.</param>
```